src/install/config/memcached.yaml

```diff
@@ -0,0 +1,87 @@
+agentName: memcached
+agentType: integration
+title: 'Memcached monitoring integration'
+metaDescription: 'Install the Memcached integration'
+introFilePath: 'src/install/memcached/intro.mdx'
+appInfo: 
+  - optionType: dropdown1
+    label: ''
+    placeholder: 'Select your environment'
+    options: 
+      - value: 'linux'
+        displayName: 'Linux'
+        logo: 'linux'
+        recommendedGuided: true
+      - value: 'windows'
+        displayName: 'Windows'
+        logo: 'windows'
+        recommendedGuided: true
+  - optionType: deployment
+    label: "Choose your install path"
+    placeholder: "Choose one:"
+    options:
+      - value: "msi"
+        displayName: "Install using MSI files (Windows)"
+        logo: 'windows'
+        recommendedGuided: true        
+      - value: "apt"
+        displayName: "Install using Apt (Debian, Ubuntu)"
+        recommendedGuided: true        
+      - value: "yum"
+        logo: 'linux'      
+        displayName: "Install using Yum (Amazon Linux, CentOS, RHEL)"
+        recommendedGuided: true        
+      - value: "zypper"
+        displayName: "Install using Zypper (SLES)"
+        recommendedGuided: true        
+        logo: 'linux'        
+      - value: "tar"
+        displayName: "Install from tarball (Other Linux distros)"  
+        recommendedGuided: true        
+        logo: 'linux'                  
+
+steps:
+  - filePath: 'src/install/memcached/appInfo.mdx'
+  - filePath: 'src/install/memcached/default-requirements.mdx'
+  - filePath: 'src/install/memcached/install-infra.mdx'
+  - filePath: "src/install/memcached/linux/install-apt.mdx"
```

Comment:
i believe live 47 is telling the branching doc to do a default path for Apt, which is messing with the doc in its neutral, reset state. Gives it all sorts of buggy behaviors

Reply:
I tried removing this in the local to see if it worked but it didn't. idk maybe you want it to do that, but am calling this out anyway

Reply:
I did intend it as we want all steps to present some sort of "default", and APT is a nice default state for it to be.